aepsych/server/server.py

```diff
@@ -297,6 +298,45 @@ def _flatten(x):
             )
         return out
 
+    def generate_experiment_table(self, experiment_id=None, table_name = 'experiment_table',
```

Comment:
With the addition of this method, there's two canonical ways of getting data out of a db. One of them is with this method, and the other is with `get_dataframe_from_replay` elsewhere in this file (which uses the replay or strat tables depending on what's available). I think that you should add a `DeprecationWarning` to the other method which recommends that users use this method instead. Other than that this PR looks pretty good!

Reply:
Thank you! ✨

I just added the deprecation warning 👀